examples/gallery/histograms/blockm.py

```diff
@@ -0,0 +1,62 @@
+"""
+Blockmean
+---------
+The :meth:`pygmt.blockmean` method allows to calculate block averages and
+to report the number of points inside each block.
+"""
+
+import pygmt
+
+# load sample data
+data = pygmt.datasets.load_japan_quakes()
+# select only needed columns
+data = data[["longitude", "latitude", "depth_km"]]
+
+# Set the region for the plot
+region = [130, 152.5, 32.5, 52.5]
+# Define spacing in x and y direction
+spacing = "150m"
+
+fig = pygmt.Figure()
+
+# ----------------------------------------------------
```

Comment:
```suggestion
```
I don't think we typically use these dashed lines in a gallery, just a tutorial.